client/src/components/previews/OrganizationPreview.js

```diff
@@ -126,36 +127,14 @@ const OrganizationPreview = ({ className, uuid }) => {
         <h4>{`Organization ${organization.shortName}`}</h4>
       </div>
       <div className="preview-section">
-        <PreviewField
-          label="Status"
-          value={Organization.humanNameOfStatus(organization.status)}
-        />
-
-        <PreviewField
-          label="Type"
-          value={Organization.humanNameOfType(organization.type)}
-        />
-
-        {renderLeadingPositions(
-          organization.positions,
-          PositionRole.LEADER.toString(),
-          pluralize(utils.titleCase(PositionRole.LEADER.humanNameOfRole()))
-        )}
-
-        {renderLeadingPositions(
-          organization.positions,
-          PositionRole.DEPUTY.toString(),
-          pluralize(utils.titleCase(PositionRole.DEPUTY.humanNameOfRole()))
-        )}
-
         <PreviewField
           label={orgSettings.longName.label}
           value={organization.longName}
         />
 
         <PreviewField
-          label={orgSettings.identificationCode?.label}
-          value={organization.identificationCode}
+          label="Type"
```

Comment:
It makes sense to keep the order of the Preview and the Show page mostly the same.

Reply:
There are sections that are shown or not the same in some form and show page or preview structures, or even if they are the same, I have adjusted them to be more comfortable and more useful to the eye.